### DISCUSSION TITLE
3D image L2 distance loss

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#5825] [Q&A] [answered]
Thanks for the great job. I am doing a 3D medical image landmark heatmap regression task. I consider using L2 distance as loss function, how should I realize it in MONAI? I found monai.metrics.MSEMetric, may be it can be used in loss function?

**Comment by KumoLiu** [comment, marked as answer]
Hi @YingdiZhang, maybe you could use `torch.nn.MSELoss`, you can find a reference in this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/3d_registration/paired_lung_ct.ipynb).

Hope it can help you, thanks!
